platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/network/OutboundConnectionManagerTest.java

```diff
@@ -73,7 +74,8 @@ void createConnectionTest() {
     @Test
     void concurrencyTest() throws InterruptedException {
         final int numThreads = 10;
-        final NodeId nodeId = NodeId.createMain(0);
+        final NodeId nodeId = new NodeId(0L);
+        ;
```

Comment:
Nit: extraneous `;` (don't feel the need to hold up the PR if this is the only change)

Reply:
removed